### ISSUE TITLE
Accessibility mode doesn't persist between pages

### ISSUE THREAD

**Issue by marksutherland** [computeranonymous/computer#106]
1) Visit a page (e.g. http://computer.twentygototen.org/)
2) Enabled accessibility mode by clicking on "White on black version"
3) Click on a link to another page on the site, e.g. a local group page
4) Observe display reverts to black on white text

This causes users to have to enable the accessibility mode on each page they visit.

**Comment by adacable** [comment]
The hack doesn't enable this to be solved without completely re-writing the site, or loads of JS. It is a minimal cost, as we only have like 15 pages. Suggest closure because of overheads.

**Comment by sjmarshy** [comment]
it wouldn't take too much JS, a few simple bits for reading/writing cookies and a couple of lines to swap styles. 

I don't mind working on this, but it'll have to be after work tonight.

**Comment by adacable** [comment]
It might be neater to append a #accessibility to every internal link? 

When I said "loads" I meant some: we've avoided it so far- we certainly shouldn't drop support for low powered smart phones/etc- so make it an enhancement rather than a drop in?

**Comment by sjmarshy** [comment]
Absolutely, if the javascript could work with the current :target hack, it would be simple enough to use it only to make the change persistent.

**Comment by tef** [comment]
Cookies. Fixed.
